src/applications/facility-locator/tests/e2e/search.cypress.spec.js

```diff
@@ -1,18 +1,25 @@
 describe('Facility Search', () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route('GET', '/v0/maintenance_windows', []);
+  });
+
   it('displays search results header after searching', () => {
```

Comment:
I would think it's worth also testing the actual flow of typing into the search and getting results that way.

Reply:
Agreed. We can beef up these tests as we move forward.